share/wake/lib/system/sources.wake

```diff
@@ -152,6 +154,7 @@ export def claimFileAsPathIn (outputDirectory: Path) (existingFile: String) (des
         def visible = outputDirectory, Nil
         def cmdline = which "cp", existingFile, desiredWorkspacePath, Nil
         makeExecPlan cmdline visible
+        | setPlanLabel "claimFileAsPathIn: cp {existingFile} {desiredWorkspacePath}"
```

Comment:
Same here, this seems like it could be simplified quite a bit (I'm not a huge fan of using Wake function names as label prefixes, but definitely not strongly opposed even so).
```suggestion
        | setPlanLabel "claim: {existingFile} -> {desiredWorkspacePath}"
```
```suggestion
        | setPlanLabel "cp: {existingFile} -> {desiredWorkspacePath}"
```